meetings/ui/src/main/frontend/src/views/Main.vue

```diff
@@ -259,6 +274,19 @@ export default {
         this.showError(this.i18n.error_load_meetings);
       }
     },
+    async loadDefaultProperties() {
+      try {
+        const response = await fetch(constants.toolPlacement + "/config");
+        if (response.ok) {
+          const data = await response.json();
+          this.showMeetingBanner = data.showMeetingBanner;
+        } else {
+          console.error('Error loading meeting banner configuration.');
+        }
+      } catch (error) {
+        console.error('Error loading meeting banner configuration:', error);
```

Comment:
```suggestion
        console.error('Error loading default configuration:', error);
```
Let's make it more generic in case we need it for other properties in the future